### ISSUE TITLE
URGENT HELP URGENT HELP

### ISSUE THREAD

**Issue by thatsitlol** [cboulay/PSMoveService#451]
so here's everything that I've done just to try and run this program.
-Bought all the cameras, controllers, and a USB dongle, that set me back around 80 dollars already so right now I'm just in the mood to just get this thing working because I'm really really pissed at how much fucking work and effort it takes for this.
Anyways first off I had a problem with the USB dongle took me a full day to fix that. Anyways I fixed that and it's working. After that I hooked up all the controllers and cameras and it was all tracking and working perfectly fine.
Then I pulled up free pie and I think connected it to trinus vr. (Or I think I just know there's a freepie setting in trinus vr, can someone help me though to make freepie work on trinus if you could). Next thing I connect my phone and trinus says "open steam vr before the game"
I open steam vr and right off the bat it's just a black box where the settings should be, then it says not responding, that goes on for a minute then it shows steam vr. But it says there's no headset, controllers, or cameras, WHICH IS BS BECAUSE ITS ALL HOOKED UP AND EVERYTHING. Ps move service is running and everything.
I've edited many files to try and to get this work. Here's my steam directory config files, and my steam apps common steam vr resources. Here's what it also says when I open steam vr. 
I've tried everything deleting it reinstalling it, validating files, please I just want it to work.
http://imgur.com/a/j55R0
LONG STORY SHORT
I've followed all tutorials but no controller or camera or headset shows up, I don't think it's ps move service put steam vr itself, just read above

**Comment by HipsterSloth** [comment]
I'm sorry to hear about your SteamVR woes. Your best bet for for all troubleshooting questions is to post to the Google group first:

https://groups.google.com/forum/#!forum/psmoveservice

Start a thread there with the above info. Also linking to the config and text files is more helpful than images of config files.

If we determine that this is actually a bug with PSMoveService or with PSMoveSteamVRBridge then we will file an issue to track the problem.

Thank you for your patience with this process.
